data_pipelines/resources/dask_resource.py

```diff
@@ -38,19 +36,45 @@ def _provision_cluster(self, context: InitResourceContext, *args, **kwargs):
 
 
 class DaskFargateResource(DaskResource):
+    region_name: str
     n_workers: int = 4
-    region: str = "eu-north-1"
+    cluster_arn: str | None = None
+    scheduler_task_definition_arn: str | None = None
+    worker_task_definition_arn: str | None = None
+    image: str | None
+    task_role_policies: list[str] | None = None
+
+    @property
+    def arn_config_provided(self) -> bool:
```

Comment:
Think this name is a bit unclear, what about `aws_resources_provided` or something like that?